src/components/pages/OktaLoginForm.tsx

```diff
@@ -0,0 +1,45 @@
+import { Button } from '@mui/material';
+import { useEffect, useState } from 'react';
+
+import { getCsrfToken } from '@/utils/csrf';
+
+interface Props {
+  path: string;
+}
+const OktaLoginForm: React.FC<Props> = ({ path }) => {
+  // fixme - csrf should probably be in a context or other shared state
+  const [csrf, setCsrf] = useState<string>();
+  useEffect(() => {
+    const token = getCsrfToken();
+    if (token) {
+      setCsrf(token);
+    }
+  }, []);
+  if (!csrf) {
+    // FIXME- if no crsrf yet, maybe reloading the window
```

Comment:
Reloading sounds good. Before this all renders, we call [fetchCurrentUser](https://github.com/greenriver/hmis-frontend/blob/ab1190e0b09615fd46bfa7630350d15520764eb5/src/modules/auth/hooks/useAuth.tsx#L154-L165) on initial page load, which will set the csrf cookie for us. So, we should only hit line 18 if the cookie was cleared somehow.

Reply:
I was encountering this situation but maybe it was just an artifact of my local dev environment.

Reply:
I can't repro it, but I think this is a fine solution to keep in here.